doc/classes/Viewport.xml

```diff
@@ -248,6 +248,7 @@
 		</member>
 		<member name="hdr" type="bool" setter="set_hdr" getter="get_hdr" default="true">
 			If [code]true[/code], the viewport rendering will receive benefits from High Dynamic Range algorithm. High Dynamic Range allows the viewport to receive values that are outside the 0-1 range. In Godot HDR uses 16 bits, meaning it does not store the full range of a floating point number.
+			[b]Note:[/b] Requires [member usage] to be set to [code]USAGE_3D[/code] or [code]USAGE_3D_NO_EFFECTS[/code], since HDR is not supported for 2D.
```

Comment:
You should be able to reference the constants directly with `[constant Viewport.USAGE_3D]`.

Reply:
Made the required changes. Guess I could update the relevant section in [Updating The Class Reference](https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html#improve-formatting-with-bbcode-style-tags) as well.

Reply:
Ah indeed, it needs to be updated to add `[constant CONSTANTNAME]`, `[constant Class.CONSTANTNAME]`, `[enum EnumName]` and `[enum Class.EnumName]`.